### ISSUE TITLE
New Release ? 

### ISSUE THREAD

**Issue by Link-God** [jmrivas86/django-json-widget#79]
Hello! Please tell me when the [fix](https://github.com/jmrivas86/django-json-widget/issues/63 ) is expected to be released ?

**Comment by wiktorcie** [comment]
BUMP

**Comment by tadamcz** [comment]
In the meantime, as a workaround, you can add it as git dependency

e.g.

```
# pyproject.toml

[tool.poetry.dependencies]
django-json-widget = {git = "https://github.com/jmrivas86/django-json-widget.git", rev="32c6acf9dd3cf27b43e0dd38a01e2af8e6c2ae10"}
```

**Comment by TheVaan** [comment]
Bump @jmrivas86

**Comment by davidemerritt** [comment]
+1

**Comment by jmrivas86** [comment]
Good news friends. I just published the new version 2.0.0 of django-json-widget on PyPi. This version includes all the PRs that were already approved and some minor changes that I have introduced  🎉

Sorry for the delay guys, I was studying for a public examination and became a father for the second time... I didn't have much time to spare. I hope I don't take too long for future versions.

Thanks a lot 😊
